cli/fathom_web/commands/pick.py

```diff
@@ -13,14 +12,25 @@
           type=Path(exists=True, file_okay=False, writable=True, dir_okay=True, allow_dash=True))
 @argument('number', type=int)
 def main(from_dir, to_dir, number):
-    """Move a given number of random things from one directory to another.
+    """Move a given number of HTML files and any extracted resources from one directory to another.
 
     Ignore hidden files.
 
     This is useful for dividing a corpus into training, validation, and testing
     parts.
-
     """
-    files = [f for f in listdir(from_dir) if not f.startswith('.')]
+    # Make these strings into ``Path``s so they are easier to work with
+    from_dir = pathlib.Path(from_dir)
```

Comment:
Any reason not to import more deeply and save some reading?

Reply:
`click` also has a `Path` class, so I can't import `Path` here.

Reply:
Ah yes. Bummer. I'd deep-import whichever one I use more often.